suse2021-ns/common/navigation.xsl

```diff
@@ -111,14 +111,13 @@
   <!-- ===================================================== -->
   <xsl:template name="is.xref.in.samebook">
```

Comment:
I agree, this would be much nicer. Additionally, I'd suggest to add the prefix `is.` for consistency reasons. The complete name would be `is.xref.within.rootid`. This would align with the other named templates therein.